package/yast2.changes

```diff
@@ -1,4 +1,11 @@
 -------------------------------------------------------------------
+Wed Aug 26 19:25:07 UTC 2015 - mfilka@suse.com
+
+- bnc#897129
+  - protect imported data agains overwriting by read
```

Comment:
interaction with user is not my cup of tea ;-) I'll rephrase it as suggested. Thanks